README.md

```diff
@@ -1,2 +1,73 @@
 # cs-entities
 Simple CloudStack Entities Framework
+
+The library provides the convenient way of working with Apache CloudStack entities through the following mechanisms:
+1. Creating and retrieving the declared entities on/from Apache CloudStack server, such as users, accounts, virtual machines, tags.
+It based on extensible Request which simplify creating requests to CloudStack.
```

Comment:
It based on the extensible request builders which simplify creating CloudStack requests.

Reply:
ok